js/lecturers.js

```diff
@@ -0,0 +1,95 @@
+    profession: 'As the main venue for new media art production in Korea, Nabi promotes cross-disciplinary collaboration and understanding among science technology, humanities, and the arts.',
+    image: 'images/img-avator/speaker_02.png',
+  },
+  {
+    id: '03',
+    name: 'Lila tretikov',
+    area: 'Executive Director of the Wikimedia Foundation',
+    profession: 'Lila Tretikov is the Executive of the Wikimedia Foundation, the nonprofit organization that operates Wikipedia. Wikipedia is freely available in 290 languages and used by nearly half a billion people around the world every month.',
+    image: 'images/img-avator/speaker_03.png',
+  },
+  {
+    id: '04',
+    name: 'Kilnam Chon',
+    area: 'Blockchain expert',
+    profession: 'European ingetration, political democracy and participation of youth through online as her major condern, Reda’s report outlining potential changes to EU copyright law was approved by the Parliament in July',
+    image: 'images/img-avator/speaker_04.png',
+  },
+  {
+    id: '05',
+    name: 'Julia Leda',
+    area: 'President of Young Pirates of Europe',
+    profession: 'European integration, political democracy and participation of youth through online as her major condemn, Reda’s report outlining potential changes to EU copyright law was approved by the Parliament in July',
+    image: 'images/img-avator/speaker_05.png',
+  },
+  {
+    id: '06',
+    name: 'Ryan Merkley',
+    area: 'CEO of Creative Commons, ex COO of the Mozilla Foundation',
+    profession: 'Ryan had been leading open-source projects at the Mozilla Foundation such as the open-source move-ment',
+    image: 'images/img-avator/speaker_06.png',
+  },
+];
+
+const showLect = document.querySelector('#tuitornll');
+let output = ' ';
+lecturersData.forEach((lecturer) => {
+  output += `
+        <div class="lecturer-card">
+           <img src="${lecturer.image}" alt="lecturer">
+           <div class="lecturer-card-details">
+                <h3>${lecturer.name}</h3>
+                <p class="tuitor">${lecturer.area}</p>
+                <p>${lecturer.profession}</p>
+           </div>
+        </div>
+          `;
+});
+showLect.innerHTML = output;
+
+const showLecturer = document.querySelector('#tuitornli');
+const lecturer = document.querySelector('#tuitornl');
+const showMoreBtn = document.createElement('button');
+showMoreBtn.className = 'show-more';
+showMoreBtn.textContent = 'More';
+showMoreBtn.addEventListener('click', () => {
+  const moreLecturers = lecturersData.slice(2);
+  showLecturer.innerHTML += moreLecturers.map((lecturer) => `
+            <div class="lecturer-card">
+               <img src="${lecturer.image}" alt="lecturer">
+               <div class="lecturer-card-details">
+                    <h3>${lecturer.name}</h3>
+                    <p class="tuitor">${lecturer.area}</p>
+                    <p>${lecturer.profession}</p>
+               </div>
+            </div>
```

Comment:
- [ ] Please stick to the Behance design as much as possible if you see your text it's not the same as the Behance design you use `text-align: center` but in Behance design the text alignment is not center so kindly fix this issue as well :+1: check below :point_down:  the comparison between your design and Behance design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-05-27 00-01-34](https://github.com/evansnyamekye/Capstone-PR/assets/45226848/ec1b5367-8799-42e1-9e6b-88c22a86cedc)

**Behance Design**
![Screenshot from 2023-05-27 00-01-11](https://github.com/evansnyamekye/Capstone-PR/assets/45226848/6e1cd150-1d96-45d7-8887-f6ddccd1fe41)